src/SpendPermissionManager.sol

```diff
@@ -96,10 +96,10 @@ contract SpendPermissionManager is EIP712 {
     address public constant NATIVE_TOKEN = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
     /// @notice Spend permission is revoked.
-    mapping(bytes32 hash => bool revoked) internal _isRevoked;
+    mapping(bytes32 hash => bool revoked) public isRevoked;
 
     /// @notice Spend permission is approved.
-    mapping(bytes32 hash => bool approved) internal _isApproved;
+    mapping(bytes32 hash => bool approved) public isApproved;
```

Comment:
For permissions which the user has approved, but not yet landed onchain (assuming it's stored in the BE DB), what would this return?